javatools/src/main/java/org/xvm/tool/Compiler.java

```diff
@@ -323,66 +295,61 @@ && isExplicitCompiledFile(outputLoc.getName())) {
      * @param repoLib  the library repository being used for compilation
      */
     protected void injectNativeTurtle(ModuleRepository repoLib) {
-        ModuleRepository repoBuild    = extractBuildRepo(repoLib);
-        ModuleStructure  moduleTurtle = repoBuild.loadModule(Constants.TURTLE_MODULE);
+        var repoBuild    = extractBuildRepo(repoLib);
```

Comment:
The variable declarations in this method seem to have been changed to `var` for no particular reason. I certainly do not want to start a multi-week debate on the use of `var` like we have with finals. But this sort of change in a PR just adds noise for the reviewers to have to look at.  I realise this particular class has been heavily changed and on the whole this method has not added too much to the review work load, but I'd prefer not to have changes to exiting code fo no apparent benefit.